GraphFilter.py

```diff
@@ -0,0 +1,81 @@
+import networkx as nx
+import time
+
+
+class GraphFilter:
+    # Configuration starts here
+
+    # Node pubkeys that I plan to open a channel to, count these as already formed.
+    channels_to_add = [
+
+    ]
+
+    # Assumed size of the new channels, should not currently have an effect
+    channel_size = 2e6
+
+    # Node pubkeys that I plan to close all channels to, count these as nonexistent.
+    channels_to_remove = [
+
+    ]
+
```

Comment:
I think the top level comment can go
I don't like the lists of channels to add and remove being here, I think they should be in the main file.
Better if they were in the config file, but I haven't come up with a satisfactory implementation of such.

Perhaps the channel add/remove step should have its own method, called from main(). That might mean freeze() can no longer be used here, but I'm planning to move away from networkx anyway.

Reply:
Okay, 
I made it a param with a default to none